apps/files_versions/lib/Versions/INeedSyncVersionBackend.php

```diff
@@ -14,6 +14,9 @@
  * @since 28.0.0
  */
 interface INeedSyncVersionBackend {
+	/**
+	 * @return void|null|int
+	 */
 	public function createVersionEntity(File $file): void;
```

Comment:
```suggestion
	public function createVersionEntity(File $file);
```
You need to actually remove the type entirely, otherwise returning anything will fail because only void is allowed.